src/screens/DecisionDetail.js

```diff
@@ -275,6 +286,26 @@ function DecisionDetail({ proposal, actions }) {
           }
         />
       </div>
+      <MultiModal
+        visible={modalVisible}
+        onClose={() => setModalVisible(false)}
+        oonClosed={() => setModalMode(null)}
```

Comment:
I think we should do this same correction on `ProposalDetail`

```suggestion
        onClosed={() => setModalMode(null)}
```